koku/api/report/queries.py

```diff
@@ -744,16 +766,16 @@ def calculate_total(self, units_value):
             'time_scope_value',
             0
         )
-        total_filter['report_type'] = self._report_type
+        total_filter['report_type'] = getattr(self, '_report_type', 'costs')
```

Comment:
I hesitate to give this a default. The aggregates table is pretty specific to the current API types we have, so potentially bad data will be returned with that default if we forget to pass in the correct report type.